### PR TITLE
Miscellaneous mission improvements 4

### PR DESCRIPTION
**Pull request by Smittytron**
- Fixes a tile error in Soviet09
- Uses English proper title casing conventions on mission names
- Moves TRUK prereq overrides to campaign-rules.yaml
- Moves IdleHunt functions to campaign-global.lua
- Sets all mission default difficulties to Normal for consistency

### CODE REVIEW

File: mods/ra/maps/exodus/exodus.lua [hunk 1]
```diff
@@ -184,13 +184,6 @@ SendSovietParadrop = function()
 	Trigger.AfterDelay(sovietParadropTicks, SendSovietParadrop)
 end
 
-IdleHunt = function(unit)
-	Trigger.OnIdle(unit, unit.Hunt)
-	Trigger.OnCapture(unit, function()
-		Trigger.ClearAll(unit)
```
Comment (reaperrr) [inline mods/ra/maps/exodus/exodus.lua:190]: Are we sure this OnCapture part isn't needed here?

File: mods/ra/maps/allies-02/rules.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ World:
 			normal: Normal
 			hard: Hard
 			tough: Real tough guy
-		Default: easy
+		Default: normal
```
Comment (reaperrr) [inline mods/ra/maps/allies-02/rules.yaml:25]: IIRC difficulties are then displayed out of order in the dropdown (you know, normal -> easy -> hard).
I don't particularly mind, but that might have been the reason this wasn't done before.

**Review by reaperrr — DISMISSED** [review]
Otherwise LGTM

File: mods/ra/maps/allies-02/rules.yaml (hunk 2 above)
Comment (Smittytron) [reply mods/ra/maps/allies-02/rules.yaml:25]: Double checked it. Easy is still on top on the dropdown.

File: mods/ra/maps/exodus/exodus.lua (hunk 1 above)
Comment (Smittytron) [reply mods/ra/maps/exodus/exodus.lua:190]: Looks like it's there to work around players using thieves by stealing Soviet tech. I don't believe they add much to the theme of the mission so I added a commit to remove thieves from this mission.

**Comment by Smittytron** [comment]
Updated to include #19042, #19081, and #19179.

File: mods/ra/maps/allies-02/rules.yaml (hunk 2 above)
Comment (reaperrr) [reply mods/ra/maps/allies-02/rules.yaml:25]: Huh... tested some more and it seems to be a TD-exclusive issue, and actually not related to what's set as default.

Anyway, no impact on this PR then.

**Review by reaperrr — APPROVED** [review]

**Review by tttppp — APPROVED** [review]
I didn't check the tile error, but the code changes look good.

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/bd5c357abcd3fa577f2e31914737a7d59321b69f...5c95f2e0b3235c1949f6e3fc62405ba5a9fc1f99

**Pull request merged**